cime_config/machines/config_machines.xml

```diff
@@ -387,6 +387,7 @@
       <env name="HDF5_USE_FILE_LOCKING">FALSE</env>
       <env name="PERL5LIB">/global/cfs/cdirs/e3sm/perl/lib/perl5-only-switch</env>
       <env name="FI_CXI_RX_MATCH_MODE">software</env>
+      <env name="MPICH_COLL_SYNC">MPI_Bcast</env>
```

Comment:
Is this a network performance tweak?

Reply:
In this case, it is a setting for alvarez and we just want to be consistent with settings already in place for pm-cpu. After SW/HW changes, it could be good to check if even still need this work-around. But at the time, there were no negative side effects to using it.